modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+
+import {Grid, GridItem} from '@workday/canvas-kit-react/layout';
+// examples
+import {GridLayout} from './examples/Grid/GridLayout';
+import {UIExample} from './examples/Grid/UIExample';
+import {GridLayoutInteractive} from './examples/Grid/GridLayoutInteractive';
+import {GridStyle} from './examples/PropTables.splitprops.tsx';
+import {GridStyleProps} from '../lib/utils/grid';
+
+<Meta title="Components/Containers/Layout/Grid" component={Grid} />
+
+# Canvas Kit Grid
+
+`Grid` is a two-dimensional layout system for the web. It lets you lay content out in rows and
+columns [CSS Grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids).
+
+If you want to see how Flexbox and Grid can
+[work together](https://css-tricks.com/css-grid-replace-flexbox/).
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Examples
+
+### UI Layout Example
+
+This example showcases what a basic UI could look like utilzing Grid. In the below example, we have
+created four elements (`Header`, `SideBar`, `BodyContent` and `Footer`). By giving each element a
+`gridArea` name, the name can be used to organize the UI however you want. If you want more info on
+how `gridArea` works with `gridTemplateAreas`,
+[see here](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas).
+
+<ExampleCodeBlock code={UIExample} />
+
+### Rows and Columns
+
+You can nest `Grid` components to build layouts with rows and columns. You can also nest `Grid`
+components within `<Grid.Item/>` components for even more possibilities.
+
+In the below example, we have a `Grid` container. It contains nine elements with one of them being
+grey. This is a `<Grid.Item/>` component and has access to child specific props, the `<Grid/>`
+component only has access to container props. This is to show what the `<Grid.Item/>` component
+can&nbsp;do.
+
+In the layout, the `<Grid.Item/>` component (grey element), has a `Row` and `Col` button. These
+buttons are interactive and if you click the `Row` button, it will move the `<Grid.Item/>` element
+between the rows and if you click the `Col` button, it will move it through the columns. These
+buttons are updating `gridRowStart`(`grid-row-start` css grid property) and
+`gridColumnStart`(`grid-column-start` css grid property). This is to showcase the child specific
+props for the `<Grid.Item/>` component.
+
+<ExampleCodeBlock code={GridLayoutInteractive} />
+
+**This is NOT best practice for accessibility reasons and we suggest against following this. This is
```

Comment:
Unsure about the pattern we have within our docs but, what if we added a ⚠️ at the beginning on this sentence. When reading this on storybook despite it being bold, I'm finding that isn't making the warning feel like a warning

Reply:
Added!